CHANGELOG.txt

```diff
@@ -36,6 +36,7 @@ Changelog
  * Docs: Document usage of custom validation for admin form pages when using `AbstractEmailForm` or `AbstractForm` pages (John-Scott Atlakson, LB (Ben) Johnston)
  * Docs: Reword `BlogTagIndexPage` example for clarity (Clifford Gama)
  * Docs: Change title of blog index page in tutorial to avoid slug issues (Thibaud Colas)
+ * Docs: Update tutorial to use `+` icon for adding child pages to Home page (Clifford Gama)
```

Comment:
Not a big issue but in the future, there's no need to include the chnagelog entry, the core team will do this when merging.

Thanks though.